src/decoders/arw.rs

```diff
@@ -45,6 +45,14 @@ impl<'a> Decoder for ArwDecoder<'a> {
     };
     let src = &self.buffer[offset..];
 
+    // Lookup the orientation of the image to store for rotating after demosaic
```

Comment:
This code seems to be duplicated across formats. Almost all formats should use the same TIFF tag with the same interpretation so it should be possible to refactor this into a single central bit of code and then allow only the weird formats override like we already do for a few other bits of metadata.